### PR TITLE
Create Airflow pipeline

### PR DESCRIPTION
**Pull request by ko3a4ok**

### CODE REVIEW

File: l8/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+apache-airflow
+apache-airflow-providers-cncf-kubernetes
```
Comment (truskovskiyk) [inline l8/requirements.txt]: add version

File: l8/training_dag.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      in_cluster=False,
+      is_delete_operator_pod=False,
+      namespace="default",
+      volumes=[volume],
+      volume_mounts=[volume_mount],
+  )
+
+  download_data_task >> training_task >> upload_task
+
+d = training()
```
Comment (truskovskiyk) [inline l8/training_dag.py:61]: why do we need this line?

File: l8/training_dag.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      namespace="default",
+      volumes=[volume],
+      volume_mounts=[volume_mount],
+  )
+
+  upload_task = KubernetesPodOperator(
+      task_id="upload_model",
+      image="ko3a4ok/zbs:latest",
+      cmds=["python", "zbs/cli.py", "upload-model", "--path=/tmp/model"],
+      env_vars={"WANDB_API_KEY": Variable.get("wandb_api_key")},
```
Comment (truskovskiyk) [inline l8/training_dag.py:51]: nice!

File: l8/inference_dag.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+      cmds=["python", "zbs/cli.py", "upload-results", "--path=/tmp/data"],
+      env_vars={"WANDB_API_KEY": Variable.get("wandb_api_key")},
+      **default_params,
+  )
+
+  clean_storage_before_start >> [download_data_task,
+                                 load_model_task] >> inference_task >> upload_results_task
+
+
+d = inference()
```
Comment (truskovskiyk) [inline l8/inference_dag.py:72]: why do we need this line?

**Review by truskovskiyk — APPROVED** [review]

File: l8/training_dag.py (hunk 2 above)
Comment (ko3a4ok) [reply l8/training_dag.py:61]: unfortunately, `@dag` creates a generator, so we need to call the function explicitly

File: l8/inference_dag.py (hunk 4 above)
Comment (ko3a4ok) [reply l8/inference_dag.py:72]: I had the same question. So, `with Dag` and `@dag` works differently: https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/dags.html#declaring-a-dag. That's why we need to call the function explicitly.

**Pull request merged**